src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.scss

```diff
@@ -6,3 +8,29 @@ hr {
 .list-group-item {
   border: 0;
 }
+
+.align-replica-zones {
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 2em;
+  padding-right: 2em;
+}
+
+ul {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+}
+
+.align-primary-zone {
+  padding-left: 4em;
+}
+
+.set-bar-width {
+  width: 100% !important;
+}
```

Comment:
there are bootstrap classes for setting the width to 100. `w-100`
https://getbootstrap.com/docs/5.0/utilities/sizing/#relative-to-the-parent

Reply:
Thanks, Done